frontend/src/components/Pages/MissionDefinitionPage/MissionDefinitionPage.tsx

```diff
@@ -175,68 +243,78 @@ function MissionDefinitionEditButtons({ missionDefinition, updateMissionDefiniti
-                                    }
-                                />
-                                <div>
-                                    <TextField
-                                        id="compact-textfield"
-                                        label={TranslateText('Days between inspections')}
-                                        unit={TranslateText('days')}
-                                        value={inspectionFrequencyDays}
-                                        onChange={(changes: ChangeEvent<HTMLInputElement>) =>
-                                            updateInspectionFrequencyFormDays(changes.target.value)
-                                        }
-                                    />
-                                    <TextField
-                                        id="compact-textfield"
-                                        label={TranslateText('Hours between inspections')}
-                                        unit={TranslateText('hours')}
-                                        value={inspectionFrequencyHours}
-                                        onChange={(changes: ChangeEvent<HTMLInputElement>) =>
-                                            updateInspectionFrequencyFormHours(changes.target.value)
-                                        }
-                                    />
-                                </div>
-                            </StyledFormSection>
-                            <StyledButtonSection>
-                                <Button onClick={handleSubmit} variant="outlined" color="primary">
-                                    {' '}
-                                    {TranslateText('Update mission definition')}{' '}
-                                </Button>
-                                <Button onClick={() => setIsEditDialogOpen(false)} variant="outlined" color="primary">
-                                    {' '}
-                                    {TranslateText('Cancel')}{' '}
-                                </Button>
-                            </StyledButtonSection>
-                        </StyledAutoComplete>
-                    </StyledDialog>
-                </StyledFormDialog>
-            )}
-        </>
+        <StyledDialog open={true}>
+            <StyledFormCard>
+                <Typography variant="h2">{TranslateText('Edit mission definition')}</Typography>
+                {getFormItem()}
+                <StyledButtonSection>
+                    <Button onClick={handleSubmit} variant="outlined" color="primary">
+                        {' '}
+                        {TranslateText('Update mission definition')}{' '}
+                    </Button>
+                    <Button onClick={() => closeEditDialog()} variant="outlined" color="primary">
+                        {' '}
+                        {TranslateText('Cancel')}{' '}
+                    </Button>
```

Comment:
To keep the same style as other dialogs the cancel button should be on the left and the other button should not be outlined